iwyu_include_picker.cc

```diff
@@ -360,8 +348,8 @@ const IncludeMapEntry libstdcpp_symbol_map[] = {
   { "std::size_t", kPrivate, "<cwchar>", kPublic },
 };
 
-// Private -> public include mappings for GNU libc
```

Comment:
This move looks incidental, and I think the comment looks better above. Please revert.

Reply:
I moved it because `libc_include_map` is no longer strictly private -> public mappings, so now the comment refers to those mappings, not including the reexported header, which is a different kind of mapping. But I'm happy to revert that if you disagree.

Reply:
Aah, I see. With that background it makes perfect sense. Nevermind this comment.